spec/steps/login_steps.rb

```diff
@@ -0,0 +1,30 @@
+module LoginSteps
+  CORRECT_EMAIL = 'lec1k2103@gmail.com'.freeze
+  CORRECT_PASS = 'testtest'.freeze
+
+  step 'I am on login page' do
+    Home.new.navigate_to_login_page
+  end
+
+  step 'I login with :email as an email and :password as a password with :checked remember me' do |email, password, checked|
+    if checked == 'checked'
```

Comment:
if you have if/else in step definition, it is a good candidate to split the step definition to 2 ones